lib/mongo/collection.rb

```diff
@@ -350,10 +350,54 @@ def watch(pipeline = [], options = {})
     # @return [ Integer ] The document count.
     #
     # @since 2.1.0
+    #
+    # @deprecated Use count_documents or estimated_document_count instead.
     def count(filter = nil, options = {})
       View.new(self, filter || {}, options).count(options)
     end
 
+    # Get a count of matching documents in the collection.
+    #
+    # @example Get the number of documents in the collection.
+    #   collection_view.count_documents
+    #
+    # @param [ Hash ] filter A filter for matching documents.
+    # @param [ Hash ] options Options for the count command.
+    #
+    # @option opts :skip [ Integer ] The number of documents to skip.
+    # @option opts :hint [ Hash ] Override default index selection and force
```

Comment:
Document that hint requires MongoDB 3.6+.

https://docs.mongodb.com/manual/reference/command/aggregate/

Reply:
Will do; should I add this to the existing methods as well (which don't seem to document it already)?

Reply:
Ack